### ISSUE TITLE
GUI Alignment Issue. [Must Fix]

### ISSUE THREAD

**Issue by sanjaypradeep** [adigunsherif/Django-School-Management-System#13]
![image](https://user-images.githubusercontent.com/18122757/89095229-790da700-d3e9-11ea-831b-088700cbcb37.png)

**Comment by adigunsherif** [comment]
Yes yes. I am hoping to work on this when I'm less busy. Are you interested in taking this up, @sanjaypradeep

**Comment by sanjaypradeep** [comment]
hey @adigunsherif actually I wish to take but I'm with other side project work.

**Comment by adigunsherif** [comment]
I'll see to it asap.

**Comment by adigunsherif** [comment]
Alright, I'll work on it as soon as I'm available

**Comment by adigunsherif** [comment]
I have checked this page and i currently think it wont fix. We'ld rather change that page to allow invoices to be uploaded from excel sheet instead.

**Comment by sanjaypradeep** [comment]
Yeah okay. But shall make this ticket open, for tracking?

**Comment by adigunsherif** [comment]
Okay, alright

**Comment by oyerohabib** [comment]
@adigunsherif @sanjaypradeep Hello Guys, i would love to work on the issue of the GUI design if it has not been solved.


Thanks

**Comment by sanjaypradeep** [comment]
@oyerohabib glad to see your message. Please go ahead and create pull request with changes, much appreciated in advance.
